### PR TITLE
Abu 820

### PR DESCRIPTION
**Pull request by dancgray**
Add disable animation for iOS 7. Could do with being able to disable scrollTo but not sure how to replace it nicely.

Changes for Trickle and narrative to follow.

### CODE REVIEW

File: src/core/js/views/menuView.js [hunk 1]
```diff
@@ -31,7 +32,12 @@ define(function(require) {
                     $('.loading').hide();
                     $(window).scrollTop(0);
                     Adapt.trigger('menuView:ready', this);
-                    this.$el.velocity({'opacity': 1}, 'fast');
+                    var styleOptions = { opacity: 1 };
+                    if (this.disableAnimation) {
+                        this.$el.css(styleOptions)
+                    } else {
+                        this.$el.velocity({'opacity': 1}, 'fast');
```
Comment (tomgreenfield) [inline src/core/js/views/menuView.js]: Should the first argument of this call be `styleOptions`?

Comment (dancgray) [inline src/core/js/views/menuView.js]: Yep, now changed.

**Comment by tomgreenfield** [comment]
+1

**Comment by belal-almassri-spongeuk** [comment]
+1

**Comment by moloko** [comment]
+1

**Comment by oliverfoster** [comment]
If you force the duration to 0 for scroll to that'd do it.

**Comment by dancgray** [comment]
Thanks @oliverfoster, zero duration added.

**Comment by oliverfoster** [comment]
+1

**Comment by moloko** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Pull request merged**